extras/vagrantExample/src/main/vagrant/Vagrantfile

```diff
@@ -56,47 +55,29 @@ Vagrant.configure(2) do |config|
     # TODO: Eventually this version of the Rya distribution will be on maven...and then we can use the following instead
     #export RYA_EXAMPLE_VERSION=4.0.0-incubating
     export RDF4J_VERSION=2.3.1
-    export ZOOKEEPER_VERSION=3.4.5-cdh4.5.0
-
+    export ZOOKEEPER_VERSION=3.4.5-cdh5.16.1
     mavenRepoUrl=http://repo1.maven.org/maven2/
-
     echo "Updating host file with permanent ip"
     sudo sed -i 's/127.0.1.1/192.168.33.10/' /etc/hosts
     cat >> /etc/hosts <<EOF
 192.168.33.10 zoo1 zoo2 zoo3
 EOF
-
     sudo -E apt-get -qq update
-
     echo "Installing Java installer..."
-    sudo -E add-apt-repository ppa:webupd8team/java || exit $?
-    sudo -E apt-get -qq update || exit $?
-    echo debconf shared/accepted-oracle-license-v1-1 select true | \
-      sudo -E /usr/bin/debconf-set-selections
-    echo debconf shared/accepted-oracle-license-v1-1 seen true | \
-      sudo -E /usr/bin/debconf-set-selections
-    sudo mkdir --parents /var/cache/oracle-jdk8-installer || exit $?
-    echo verbose=off >> /var/cache/oracle-jdk8-installer/wgetrc || exit $?
-
-    echo "Installing Java..."
-    sudo -E apt-get -qq install -y oracle-java8-installer || exit $?
-    sudo ln --force -s /usr/lib/jvm/java-8-oracle/ /usr/lib/jvm/default-java
-
+    sudo -E apt-get install openjdk-8-jre
```

Comment:
jdk v8 is not officially supported on the version of ubuntu used by this image:
  https://askubuntu.com/questions/464755/how-to-install-openjdk-8-on-14-04-lts/944260#944260
Looking for input on this.  I think we should use a newer Ubuntu LTS version.  I am trying "bento/ubuntu-18.04"